scripts/ci_run_test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -x
+
+TEST_RESULT_FILE="/root/project/test_results.xml"
+
+# copy unity license file
+mkdir /root/.local/share/unity3d/Unity
+cp /root/project/Unity_license.ulf /root/.local/share/unity3d/Unity/Unity_lic.ulf
+
+# login to gmsaas (genymotion tool), and create device instance for tests
+gmsaas auth login $GENY_USERNAME "$GENY_PASSWORD"
+instance_id=$(gmsaas instances start 107d757e-463a-4a18-8667-b8dec6e4c87e ci_test)
```

Comment:
```107d757e-463a-4a18-8667-b8dec6e4c87e``` ?

Reply:
restructure to const and add documentation